src/app/tests/integration/chip_im_initiator.cpp

```diff
@@ -698,7 +698,11 @@ int main(int argc, char * argv[])
     InitializeChip();
 
     err = gTransportManager.Init(chip::Transport::UdpListenParameters(&chip::DeviceLayer::InetLayer)
+#if INET_CONFIG_ENABLE_IPV4
                                      .SetAddressType(chip::Inet::kIPAddressType_IPv4)
+#else
+                                     .SetAddressType(chip::Inet::kIPAddressType_IPv6)
+#endif
```

Comment:
I feel like this should be runtime configured in the example, separate from whether support to runtime configure is built. Suggest a TODO. It is odd to me that a host-based test would need to be rebuilt to support IPv4/v6

Reply:
I wanted to make it IPv6 by default, however cirque hardcodes IPv4 so I cannot do that.

I agree that this should be different, however if long term is to have cirque use IPv6 exclusively I would prefer to keep this and delete the ipv4 stuff  later.

Reply:
Will try to use the input IP address type and see if that passes CI. As long as the deltas are small, I am willing to update.

if the deltas start asking for argument parsing, then I prefer to remove this at the same time as cirque update to only use ipv6

Reply:
Updated initiator/requester to use input IP address type.

Responder still remained compile time, which is not ideal, however I expect cirque update to take care of this not the current PR. The current PR ensures compilation is possible, does not fix all issues.